### PR TITLE
Restored telekinesis functionality

### PR DESCRIPTION
**Pull request by JustSumBody**
#10606

Items can be TK grabbed again
Also fixed up the formatting of the file because wow
Air alarms, fire alarms, and intercoms still cannot be interacted with(a potential issue that existed long before this grab bug which I'm having trouble figuring out). For now, this will at least fix the grab though.

### CODE REVIEW

**Comment by PJB3005** [comment]
The items which can't be interacted with is due to their code being awful and doing ghetto sanity instead of using standardized ways.

File: code/_onclick/telekinesis.dm [hunk 1]
```diff
@@ -81,145 +79,142 @@ var/const/tk_maxrange = 15
...
+/obj/item/tk_grab/dropped(mob/user as mob)
+	if(focus && user && loc != user && loc != user.loc) // drop_item(null, ) gets called when you tk-attack a table/closet with an item
+		if(focus.Adjacent(loc))
+			focus.loc = loc
+	qdel(src)
 
 
 	//stops TK grabs being equipped anywhere but into hands
-	equipped(var/mob/user, var/slot, hand_index)
-		if(!hand_index)	return
```
Comment (JustSumBody) [inline code/_onclick/telekinesis.dm]: This is what broke everything

File: code/_onclick/telekinesis.dm [hunk 2]
```diff
@@ -81,145 +79,142 @@ var/const/tk_maxrange = 15
...
-		qdel(src)
```
Comment (JustSumBody) [inline code/_onclick/telekinesis.dm:96]: This is what broke everything

**Comment by clusterfack** [comment]
Someone made a runtime with all the types of wires

**Pull request merged**